### ISSUE TITLE
Difference between OPENAI_KEY and OPENAI_API_KEY and how to put SERPAPI_API_KEY

### ISSUE THREAD

**Issue by gitfans5** [dair-ai/Prompt-Engineering-Guide#25]
What is the difference between OPENAI_KEY and OPENAI_API_KEY in "os.environ["OPENAI_API_KEY"] = os.getenv("OPENAI_KEY")"?

Should one input the SERPAPI_API_KEY inside the quotation mark in "os.environ["SERPAPI_API_KEY"] = os.getenv("SERPAPI_API_KEY")"

Thanks.

**Comment by omarsar** [comment]
Hi @gitfans5 You need an API Key for both of these services. 

You can set up accounts and try to get your API keys.  

For SerpAPI: https://serpapi.com/users/sign_in
For OpenAI API: https://platform.openai.com/overview

**Comment by ucas010** [comment]
hi ,dear
in the [notebook](https://github.com/dair-ai/Prompt-Engineering-Guide/blob/main/notebooks/pe-lecture.ipynb), should I have a OPENAI_API_KEY for running the rp?
`Make sure to replace OPENAI_API_KEY with your own key.`

**Comment by omarsar** [comment]
@ucas010 Yes, you have to use your own OPEN_API_KEY. You will need to create an account with OpenAI(https://platform.openai.com/overview).

**Comment by dr-nefardio3058** [comment]
Hi there! We wanted to check in and see if the issue you reported earlier has been resolved. If it has, we would greatly appreciate it if you could take a moment to close the issue. It's super easy – just click the "Close Issue" button at the bottom of the issue page.

Keeping the issue tracker clean and tidy is important for the health of our codebase. With fewer issues to manage, we can focus on the ones that need our attention and ensure that our code remains maintainable in the long run. For example, it's easier to identify trends and common issues when there are fewer open issues, which can lead to more efficient bug fixing and better overall code quality.

Thanks so much for your help in keeping our repo organized and contributing! 😄
